web_widget_darkroom/static/src/js/widget_darkroom.js

```diff
@@ -0,0 +1,224 @@
+/**
+*    Copyright 2013 Matthieu Moquet
+*    Copyright 2016-2017 LasLabs Inc.
+*    License MIT (https://opensource.org/licenses/MIT)
+**/
+
+odoo.define('web_widget_darkroom.darkroom_widget', function(require) {
+    'use strict';
+
+    var core = require('web.core');
+    var common = require('web.form_common');
+    var session = require('web.session');
+    var utils = require('web.utils');
+    var _ = require('_');
```

Comment:
This line can be dropped. It doesn't work in v10, and `_` should just be available as a global.